features/travel_advice.feature

```diff
@@ -23,3 +23,8 @@ Feature: Travel Advice
       | Path                                   |
       | /foreign-travel-advice.atom            |
       | /foreign-travel-advice/luxembourg.atom |
+
+  @normal
+  Scenario: Country feed should contain correct website root
+    When I visit "/foreign-travel-advice/ireland.atom"
+    Then the XML ID should be correct
```

Comment:
'should be correct' could be more specific, e.g. 'links to the correct website root'?